webapp/templates/team/team_editor.html.twig

```diff
@@ -31,6 +31,40 @@
 
 {% block content %}
 
+    {% if current_team_contest is not empty and not current_team_contest.allowSubmit %}
+        <div data-flash-messages>
+            {% include 'partials/alert.html.twig' with {'type': 'danger', 'message': 'Submissions disabled.\nPlease switch to the active contest using trophy icon in the top navigation.'} %}
+
+            {% set available_contests_cids = current_team_contests | keys %}
+            {% set problem_available_contests = problem.getContestProblems | filter(
+                contest_problem => contest_problem.contest.cid in available_contests_cids and contest_problem.contest.allowSubmit
+            ) %}
+
+            <div class="alert alert-info alert-dismissible fade show" role="alert">
+                {% if problem_available_contests is not empty %}
+                    This problem is now available in the following contests
+
+                    <ul class="list-unstyled ml-4 mt-1 mb-1">
+                        {% for contest_problem in problem_available_contests %}
+                            {% set contest = contest_problem.contest %}
+                            <li>
+                                <a href="{{ path('team_problems', {'cid': contest.cid}) }}">
+                                    <i class="fas fa-book-open"></i> {{ contest.name }}
+                                </a>
+                            </li>
+                        {% endfor %}
+                    </ul>
```

Comment:
tohle je hodne nice. jeste detail, jestli nechceme misto knizky pouzit trophy ikonku?

Reply:
ono te to bere primo na ten problemset, ktery pouziva vsude knizku, tak jsem se drzel toho

Reply:
joo, jasne